modules/sql-engine/src/test/resources/mapping/correlated.test

```diff
@@ -129,11 +129,10 @@ Fragment#0 root
   exchangeSourceNodes: {1=[N1]}
```

Comment:
I think we also need mapping tests for HashJoin (similar to merge join ones).

Reply:
done